### ISSUE TITLE
Request dark window decoration when Lagrange is set to dark UI theme

### ISSUE THREAD

**Issue by despens** [skyjake/lagrange#549]
On Linux, Lagrange uses the default window decoration to provide a nice top title bar. On distributions using Gnome, like Ubuntu, Fedora, etc, that element's color is light grey, unless the user has switched to dark mode. In Lagrange's beautiful dark UI, this looks not so nice and even sometimes makes it hard to visually identify the window title bar. It would be great if Lagrange would request the dark version of the window decoration when set to dark UI.

This doesn't look so good:
![image](https://user-images.githubusercontent.com/571494/198956643-b8df6bf7-3fdc-4cc6-a52d-357ffc5f7e9c.png)

This looks good:
![image](https://user-images.githubusercontent.com/571494/198956698-6b21ef0f-c468-47d4-b7e7-8202de996a8b.png)

**Comment by skyjake** [comment]
Is there a GNOME API that allows such requests? Lagrange currently has no direct integration with GNOME whatsoever, since everything goes through SDL, and that library does not seem to support requesting this.

If it is an X11 window manager API/property, that is likewise currently not available to the app.

**Comment by despens** [comment]
I see, that's fine.
As a workaround I am now running the script below. It allows to interactively pick a window that then is switched to dark theme variant in Gnome. Work with other applications as well.

```bash
#!/bin/bash
xdotool selectwindow | xargs -r xprop -f _GTK_THEME_VARIANT 8u -set _GTK_THEME_VARIANT "dark" -id
```

**Comment by skyjake** [comment]
Thanks! I'll see if I can have the equivalent of that command built into the app.

**Comment by skyjake** [comment]
This has now been implemented in the "work/v1.15" branch. It sets the window's _GTK_THEME_VARIANT property according to the currently chosen app UI color theme.
